### PR TITLE
Bugfix: 28515

### PR DESCRIPTION
**Pull request by giancorzo**
I used gravatar to get user profile image, for the MVP we aren't storing user images from Auth0.

**Note**: should I change header avatar image to get the same image everywhere or header avatar should continue using Auth0?

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
to answer your question, because the header image is _always_ going to be the current user, we can just use the auth0-provided image.

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/index.tsx [hunk 1]
```diff
@@ -3,9 +3,10 @@ import { compose } from 'recompose';
 import { withTranslations, i18nProps } from '@lib/i18n';
 import { withData } from '../edit/with-data';
 import { UserAttributes } from '@data/models/user';
-import { getPictureUrl } from '@lib/auth0';
 import { attributesFor } from '@data';
 
+import md5 from 'md5-hash';
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/index.tsx]: I wish there were more rules for imports, but generally, I like to have thing sorted like this:
```
npm-package imports

top-level imports, sorted alphabetically

relative imports, sorted alphabetically
```
(with line breaks separating each section)

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/index.tsx [hunk 2]
```diff
@@ -29,12 +30,14 @@ class User extends React.Component<IProps> {
     const phone = user.phone ? user.phone : t('profile.noPhone');
     const timezone = user.timezone ? `(${user.timezone})` : t('profile.noTimezone');
 
+    const gravatarHash = md5(user.email.trim().toLowerCase());
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/index.tsx]: what if a user doesn't have an email?

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/index.tsx [hunk 3]
```diff
@@ -29,12 +31,14 @@ class User extends React.Component<IProps> {
     const phone = user.phone ? user.phone : t('profile.noPhone');
     const timezone = user.timezone ? `(${user.timezone})` : t('profile.noTimezone');
 
+    const gravatarHash = md5((user.email || '').trim().toLowerCase());
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/index.tsx]: what does the gravatar look like without an email?

Comment (giancorzo) [reply source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/index.tsx]: Gravatar when not email or bad hash is provided we get a default image. Checkout the param of the url, param d is for default

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/__tests__/acceptance-test.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+          }
+        }]
+      });
+    });
+
+    beforeEach(async function () {
+      await visit('/users/1');
+    });
+
+    it('shows phone and timezone',() => {
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/__tests__/acceptance-test.tsx]: It's important to test the negative case as well. 

Can't test only the happy path

File: source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/__tests__/page.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import {
+  interactor, isPresent
+} from '@bigtest/interactor';
+
+@interactor
+export class UserInteractor {
+  constructor(selector?: string) { }
+
+  name = isPresent('[data-test-show-profile-name]');
```
Comment (NullVoxPopuli) [inline source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/__tests__/page.ts]: These names don't semantically represent Boolean values

**Pull request merged**